modules/tinymce/src/plugins/lists/main/ts/Plugin.ts

```diff
@@ -10,13 +10,17 @@ import * as Api from './api/Api';
 import * as Commands from './api/Commands';
 import * as Keyboard from './core/Keyboard';
 import * as Buttons from './ui/Buttons';
+import { hasRtcPlugin } from './core/DetectRtc';
+
+export default () => {
+  PluginManager.add('lists', (editor) => {
+    if (hasRtcPlugin(editor) === false) {
```

Comment:
Depending on how the API ends up, this plugin may return but generally it's all baked into the RTC model